engine/runtime/src/main/scala/org/enso/compiler/pass/desugar/SectionsToBinOp.scala

```diff
@@ -146,21 +145,16 @@ case object SectionsToBinOp extends IRPass {
           )
 
         } else {
-          val opCall = Application.Prefix(
+          Application.Prefix(
             function             = op,
-            arguments            = List(arg, rightCallArg),
+            arguments            = List(arg),
             hasDefaultsSuspended = false,
-            location             = None,
+            location             = loc,
             passData,
             diagnostics
           )
-
```

Comment:
This is the fix?

>  SectionsToBinOp compiler pass produces function application for left sections

What source code this change affects? The expression...

> `(1 +)`

...will now become `Application.Prefix(function="+", arguments=[1])`. OK, that seems correct.